sdk/communication/azure-communication-administration/README.md

```diff
@@ -87,12 +89,157 @@ communicationIdentityClient.revokeTokens(user, OffsetDateTime.now());
 ### Deleting a user
 Use the `deleteUser` function to delete a user.
 
-<!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L114-L115 -->
+<!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L120-L121 -->
 ```java
 // delete a previously created user
 communicationIdentityClient.deleteUser(user);
 ```
 
+## Phone plans overview
```

Comment:
Should we move the phone plans overview under the overall "Key Concepts" section? Under "Key Concepts" we could have subsections for Identity and Phone Numbers.